.circleci/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# Force-push the built HTML to the `gh-pages` branch.
+#
+
+set -e
```

Comment:
`set -eu` is a safer default.

I usually check my bash scripts with [ShellCheck](https://github.com/koalaman/shellcheck)

The changes for fixing the warnings are here: https://gist.github.com/iperdomo/dbdfa79e7b5cd77e90bcce469507a7da

Reply:
Fixed in commit 673b5ddc7e35162c01ef8cda562cb2d8fcef3b1a.